Sources/MapboxMaps/Foundation/Extensions/Encodable+ToJSON.swift

```diff
@@ -16,4 +16,13 @@ internal extension Encodable {
         // on `first` being non-nil
         return array.first!
     }
+
+    /// Given an Encodable object return the JSON representation as a string
+    /// - Throws: Errors occurring during conversion.
+    /// - Returns: A string with JSON representing the object.
+    func toString() throws -> String {
+        let data = try JSONEncoder().encode(self)
+
+        return String(decoding: data, as: UTF8.self)
+    }
```

Comment:
I personally don't like the idea to have a method like that where JSONEncoder will be instantiated. 
The code is trivial and has no options to manipulate JSONEncoder properties.
However, I'm not going to insist on it.

Reply:
Any alternatives in mind? There's no particular significance in using `JSONEncoder` indeed - it just was the easiest way to obtain a string from an object.